nucypher/policy/collections.py

```diff
@@ -250,7 +253,7 @@ def __init__(self, blockchain_signature=NOT_SIGNED, *args, **kwargs):
 
     @classmethod
     def splitter(cls):
-        return BytestringKwargifier(cls,
+        return NucypherBSSKwargifier(cls,
                                     blockchain_signature=65,
```

Comment:
We now have a constant for this, `LENGTH_ECDSA_SIGNATURE_WITH_RECOVERY`, in blockhain/eth/constants